.github/CODEOWNERS

```diff
@@ -1,2 +0,0 @@
-# All files
-*			@envoyproxy/gateway-maintainers
```

Comment:
doesnt this need to be set, so GH can cc these users on PRs ?

Reply:
I think we only need this in the root of the Repo.

Reply:
can you link the relevant GH docs that support this ?

Reply:
CODEOWNERS often keeps in the root of the project, this PR added one in https://github.com/envoyproxy/gateway/pull/1156/files#diff-fcf14c4b7b34fe7a11916195871ae66a59be87a395f28db73e345ebdc828085bR3. So we do not need the CODEOWNERS under .github/